parts/xmem_nozzle.go

```diff
@@ -2019,13 +2028,15 @@ func (xmem *XmemNozzle) updateSystemXattrForTarget(wrappedReq *base.WrappedMCReq
 			case base.XATTR_MOBILE:
 				if setOpt.preserveSync {
 					// Skip the source _sync value
+					xmem.RaiseEvent(common.NewEvent(common.SourceSyncXattrRemoved, nil, xmem, nil, nil))
 					continue
 				}
 			case base.XATTR_IMPORTCAS:
 				if meta != nil && !meta.IsImportMutation() {
 					// Skip it if it is no longer an import mutation. This happens when import mutation is mutated again.
 					continue
 				}
+				xmem.RaiseEvent(common.NewEvent(common.ImportMutationsSent, nil, xmem, nil, nil))
```

Comment:
If we follow my original suggestion, you won't need to add a new ImportMutationSent. 
Instead, you can consolidate the logic at Xmem raising "DataSent" event, with the "additional" portion.

Reply:
Done